Packs/MandiantAdvantageThreatIntelligence/Integrations/MandiantAdvantageThreatIntelligence/README.md

```diff
@@ -0,0 +1,2478 @@
+Enrich Indicators of Compromise, and fetch information about Actors, Malware Families, and Campaigns from Mandiant Advantage.
+This integration was integrated and tested with version xx of Mandiant Advantage Threat Intelligence
```

Comment:
```suggestion
This integration was integrated and tested with version x.x of Mandiant Advantage Threat Intelligence
```
@chrishultin - Please replace here the `x.x` with the version of Mandiant that you tested the integration with